posts/2021-04-30-dev-mode-with-aot.adoc

```diff
@@ -0,0 +1,73 @@
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Open Liberty development mode startup improved with AOT
+Irwin D'Souza <https://github.com/dsouzai>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+// // // // // // // //
+// Above:
+// Do not insert any blank lines between any of the lines above.
+//
+// "open-graph-image" is set to OL logo. Whenever possible update this to a more appriopriate/specific image (For example if present a image that is being used in the post). However, it
+// can be left empty which will set it to the default
+//
+// Replace TITLE with the blog post title.
+// Replace AUTHOR_NAME with your name as first author.
+// Replace GITHUB_USERNAME with your GitHub username eg: lauracowen
+// Replace DESCRIPTION with a short summary (~60 words) of the release (a more succinct version of the first paragraph of the post).
+//
+// Replace AUTHOR_NAME with your name as you'd like it to be displayed, eg: Laura Cowen
+//
+// Example post: 2020-04-02-generate-microprofile-rest-client-code.adoc
+//
+// If adding image into the post add :
+// -------------------------
+// [.img_border_light]
+// image::img/blog/FILE_NAME[IMAGE CAPTION ,width=70%,align="center"]
+// -------------------------
+// "[.img_border_light]" = This adds a faint grey border around the image to make its edges sharper. Use it around screenshots but not           
+// around diagrams. Then double check how it looks.
+// There is also a "[.img_border_dark]" class which tends to work best with screenshots that are taken on dark
+// backgrounds.
+// Change "FILE_NAME" to the name of the image file. Also make sure to put the image into the right folder which is: img/blog
+// change the "IMAGE CAPTION" to a couple words of what the image is
+// // // // // // // //
+
+Open Liberty development mode, or dev mode, is a feature in Open Liberty that enhances the developer’s experience by providing hot reload and deployment, on-demand testing, and debugger supportfootnote:[https://openliberty.io/blog/2019/10/22/liberty-dev-mode.html]. When using a JDK with the Eclipse OpenJ9 VM to run Open Liberty, the VM feature that is used to enable dev mode is called Full Speed Debug (FSD). One of the biggest benefits of OpenJ9 is its fast startup; this is in big part due to the Shared Classes Cache (SCC) feature, which contains both Classes and previously (dynamically) compiled AOT Codefootnote:[https://www.eclipse.org/openj9/docs/shrc/]. Before Eclipse OpenJ9 Version 0.26.0, the VM did not have AOT support for FSD; this meant that Open Liberty dev mode could not benefit from the fast startup that normally comes with Eclipse OpenJ9. However, starting from version 0.26.0, FSD support has been added for AOT compilations, and therefore Open Liberty dev mode does see the boost in startup from the SCC.
```

Comment:
AOT should be spelled out on first mention and should provide a 1-2 sentence explanation of the concept

Reply:
Made requested changes (see https://github.com/OpenLiberty/blogs/compare/ed1c305e8fee72472f81e224a1a82d205afd5bab..83e09da74325b699dcaa22fe68b54e5f35ead76f)